### PR TITLE
Improve search suggestions

### PR DESCRIPTION
**Pull request by ZeeshaanMaudar**
Add More Results and styled

### CODE REVIEW

File: assets/js/biz-portal.js [hunk 1]
```diff
@@ -35,12 +35,16 @@ $("#custom-templates .typeahead").typeahead(null, {
...
   source: businesses,
+  limit: 5,
   templates: {
     empty: [
       '<div class="empty-message">',
       "Business not found",
       "</div>"
     ].join("\n"),
+    footer : [
+      '<a class="search-result-links" href="http://192.168.99.100:8000/bla"><p class="more-results-search">More results</p></a>'
```
Comment (jbothma) [inline assets/js/biz-portal.js]: something like `/businesses/?q=$(input).value`

File: assets/js/biz-portal.js [hunk 2]
```diff
@@ -23,26 +23,26 @@ var businesses = new Bloodhound({
...
+  displayKey: "registered_name",
   source: businesses,
   templates: {
     empty: [
       '<div class="empty-message">',
       "Business not found",
       "</div>"
     ].join("\n"),
+    footer : (context) => (
+      `<a class="search-result-links" href="/businesses/?q=${context.query}"><p class="more-results-search">More results</p></a>`),
```
Comment (jbothma) [inline assets/js/biz-portal.js]:
```suggestion
      `<a class="search-result-links" href="/businesses/?q=${context.query}">
         <p class="more-results-search">See all results</p>
       </a>`),
```

File: assets/js/biz-portal.js (hunk 1 above)
Comment (ZeeshaanMaudar) [reply assets/js/biz-portal.js]: Covered

File: assets/js/biz-portal.js (hunk 2 above)
Comment (ZeeshaanMaudar) [reply assets/js/biz-portal.js]: Done

**Pull request merged**